### PR TITLE
#37 add trainee onboarding section

### LINKED ISSUE #37: Add trainee on-boarding section
This is a section for trainees/guests to consult before joining us so they know about our structure, working practcies and schedules, and what on boarding would look like for them.

It could also contain a link to the onboarding questionnaire where they can tell us about thier programming skills, aims, interests etc

- meet face to face
- local tour
- arrange 1-2-1s with team
- invite to meetings
- invite MT-CSC team
- primary and secondary supervisor nominated
- weekly 1-2-1 with primary supervisor

### PR DESCRIPTION
**Pull request by AnilMistry**
Resolves #37

### CODE REVIEW

File: _handbook/onboarding.md [hunk 1]
```diff
@@ -31,19 +38,69 @@ Street.
...
 
 **Electronic Staff Record**
 
 In order to check payslips, p60, pension contributions etc. you need to get access to your Electronic Staff Record.
    - Browse https://my.esr.nhs.uk/dashboard/web/esrweb
    - Go to login to ESR
    - Click Unlock Account if is your first time
-   - Fill in details and wait for an email with instructions. It will usually be your username and a link to set up your password
+   - Fill in details and wait for an email with instructions. It will usually be your username and a link to set up 
+   - your password
```
Comment (hshuaib90) [inline _handbook/onboarding.md]: Is this typo? Not sure why new line

File: _pages/getinvolved.md [hunk 2]
```diff
@@ -8,29 +8,43 @@ If what we do sounds exciting, get involved and work with us! There are several
 
 ### Temporary Posts
 ##### Scientists
-- Interested applicants can apply through any one of the several options available as part of the National School of Healthcare Science (NSHCS) Scientist Training Programme (STP). The NSHCS STP allows trainees to allocate part of the programme time to working with us and we've summarised the options and the different timeframes below. For the most up-to-date information, we recommend applicants refer to the [NSHCS STP page](https://www.nshcs.hee.nhs.uk/programmes/stp/) and our dedicated [STP page](../stp.html). 
+- Interested applicants can apply through any one of the several options available as part of the National School of 
+- Healthcare Science (NSHCS) Scientist Training Programme (STP). The NSHCS STP allows trainees to allocate part of the
```
Comment (hshuaib90) [inline _pages/getinvolved.md]: Again, probably a typo, not a new line

File: _pages/stp.md [hunk 3]
```diff
@@ -31,32 +31,52 @@ Trainees who specialise in Clinical Scientific Computing will gain a variety of
...
 The curriculum and in particular the rotations now have a greater emphasis on reflective practice. We discuss reflective practice in the context of IACC which will also support you preparing for rotations and we’ll be providing more resources designed to support the rotations soon.
 
+### Our commitment to STP training
+
+We STP trainers and supervisors completed the STP programme ourselves, so we're acutely aware of it challenges. 
+We're committed to continuously improving the experience for each new STP trainee. We will: 
+
+- Confirm the trainee's STP requirements at the start of the rotation/placement.
+- Assign useful CSC tasks; no fake work for the sake of an exercise.
+- Undertake addition [Educator training](https://portal.e-lfh.org.uk/myElearning/Index?HierarchyId=0_54104_54116&programmeId=54104)
```
Comment (hshuaib90) [inline _pages/stp.md]: typo: additional

**Review by hshuaib90 — CHANGES_REQUESTED** [review]
Thanks Anil, this is great. Just a couple of typos. Also what's with the image? Can we get a more specific one?

**Comment by AnilMistry** [comment]
I've also removed the photo, until we have a better policy on photo themes!

**Review by hshuaib90 — APPROVED** [review]

**Pull request merged**